frontend/src/main/pages/ProfilePage.js

```diff
@@ -1,18 +1,47 @@
-import { Row, Col } from "react-bootstrap";
+import { Row, Col, Button, Form } from "react-bootstrap";
 import RoleBadge from "main/components/Profile/RoleBadge";
 import { useCurrentUser } from "main/utils/currentUser";
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import { toast } from "react-toastify";
+import { useForm } from "react-hook-form";
+import { useBackendMutation } from "main/utils/useBackend";
+import UsersTable from "main/components/Users/UsersTable";
 
-import { Inspector } from "react-inspector";
 const ProfilePage = () => {
   const { data: currentUser } = useCurrentUser();
+  const { root } = currentUser || {};
+  const { user } = root || {};
+  const { email, pictureUrl, fullName} = user || {};
 
-  if (!currentUser.loggedIn) {
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+  } = useForm();
+
+  const objectToAxiosParams = (user) => ({
+    url: "/api/currentUser/updateAlias",
+    method: "POST",
+    params: {
+      proposedAlias: user.proposedAlias,
+    },
+  });
+  const onSuccess = (user) => {
+    toast(`Alias Awaiting Moderation: ${user.proposedAlias}`);
+  };
+
+  const mutation = useBackendMutation(objectToAxiosParams, { onSuccess }, ["current user"]);
+  
+  if (!currentUser?.loggedIn) {
```

Comment:
```suggestion
  if (!currentUser.loggedIn) {
```

I don't think you need this if you adjust the test properly (whatever problem this sovled, there's a better solution).  And introducing it adds an extra mutation you have to cover.   So better to leave it out.